about.html

```diff
@@ -3,14 +3,143 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200;400;500;600&display=swap" rel="stylesheet">
   </head>
-  <body>
+  <body class="margin-t-l" >
     <nav class="nav-bar">
-      <a href="index.html">Home</a>
-      <a href="">Education</a>
+      <div class="layout-container">
+      <div class="align-right"> 
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="">Projects</a>
+        <a href="">Contacts</a>
+      </div>
+    </div>
     </nav>
-    <h1>About</h1>
+
+    <main class="container  layout-container">
+      <div class="main-left">
+        <img src="./Pictures/ProfilePicture.jpeg" alt="Profile Picture" class="img-profile" />
+        <div>
+          <p>e-mail</p>
+          <p>telefono</p>
+          <p>facebook</p>
+        </div>
+      </div>
+      <div class="main-right">
+        <h1>Giorgio Carrara</h1>
+        <p>
+          is simply dummy text of the printing and typesetting industry. Lorem
+          Ipsum has been the industry's standard dummy text ever since the
+          1500s, when an unknown printer took a galley of type and scrambled it
+          to make a type specimen book. It has survived not only five centuries,
+          but also the leap into electronic typesetting, remaining essentially
+          unchanged.
+        </p>
+        <p>
+          Contrary to popular belief, Lorem Ipsum is not simply random text. It
+          has roots in a piece of classical Latin literature from 45 BC, making
+          it over 2000 years old.
+        </p>
+      </div>
+    </main>
+
+  <div class="layout-container">
+
+    <div class="layout-container back-color">
+      <h2>Languages and Tools</h2>
+    </div>
+    
+  
+
+  <div class="table-tools">
```

Comment:
Two divs wrapping here is not necessary. If you remove one of them it won't have impact on your layout. Especially because they have pretty much the same styling.